atqa-framework/browserCaps.js

```diff
@@ -0,0 +1,52 @@
+const initBrowser = class BrowserCaps {
```

Comment:
`module.exports = class ...` doesn't work? Regarding OOP, no idea what's the benefit here. What kinds of extended logic it can have?

Reply:
> `module.exports = class ...` doesn't work? Regarding OOP, no idea what's the benefit here. What kinds of extended logic it can have?

Many instances of args to run 100 same browsers with different caps - I know I can emulate such logic with prototypes (and it always will be a prototype in js to be honest) but then...what difference? we have a OOP syntax sugar as class so let's use it why not? 

I just rewrite this class to use spread operator and make initCaps method as static. For it looks good now.